src/frontend/src/pages/ProjectPage/pages/ContractPage/components/RequestDetailsSideSheet/CompactPersonDetails.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { PersonCard } from '@equinor/fusion-components';
+import Personnel from '../../../../../../models/Personnel';
+import * as styles from './styles.less';
+import AzureAdStatusIcon from '../../pages/ManagePersonnelPage/components/AzureAdStatus';
+
+type CompactPersonDetailsProps = {
+    personnel: Personnel;
+};
+const CompactPersonDetails: React.FC<CompactPersonDetailsProps> = ({ personnel }) => {
+    return (
+        <div className={styles.compactPersonDetails}>
+            <PersonCard personId={personnel.azureUniquePersonId} />
+            <div className={styles.textField}>
+                <span className={styles.title}>{'AD status'}</span>
+                <span className={styles.content}>
+                    {AzureAdStatusIcon(personnel.azureAdStatus || 'NoAccount')}
+                </span>
+            </div>
+            <div className={styles.textField}>
+                <span className={styles.title}>{'Disciplines'}</span>
+                <div className={styles.content}>
+                    {personnel.disciplines.map(discipline => (
+                        <div className={styles.disciplineChip}>
```

Comment:
There's a Chip component in the fusion components

Reply:
The Chip component was buggy, will come up with a fix and replace it